tables/idxutils.py

```diff
@@ -471,7 +471,7 @@ def nextafter(x, direction, dtype, itemsize):
     direction."""
 
     assert direction in [-1, 0, +1]
-    assert dtype.kind == "S" or type(x) in (bool, int, float)
+    assert dtype.kind == "S" or type(x) in (bool, int, long, float)
```

Comment:
fixed it for PY3 (using six.integer_types)... sorry -- should have kept it in mind

as for **why** -- didn't look in detail, but if interested, could do... let me know